docs/experimental_features/modin_in_the_cloud.rst

```diff
@@ -0,0 +1,68 @@
+Modin in the Cloud
+==================
+
+The modin implements functionality that allows to transfer computing to the cloud.
+Please note that this feature is experimental and behavior or interfaces could be changed.
+
+Prerequisites
+"""""""""""""
+
+  * Sign up with a cloud provider and get credentials file. Currently supported only AWS.
+  (For AWS, on credentials file format see https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-where)
+
+Setup environment
+"""""""""""""""""
+pip install modin[remote] - which install the following dependencies:
+  * RPyC - https://github.com/tomerfiliba-org/rpyc
+  Allows to perform remote procedure calls.
+  * Cloudpickle - https://github.com/cloudpipe/cloudpickle
+  Allows pickling of functions and classes, which is used in our distributed runtime.
+  * Boto3 - https://github.com/boto/boto3
+  Allows to create and setup AWS cloud machines. Optional library for Ray Autoscaler.
+
+It also needs Ray Autoscaler component, which implicitly installed with Ray.
+More information in https://docs.ray.io/en/releases-0.8.2/autoscaling.html#automatic-cluster-setup.
+
+Architecture
+""""""""""""
+
+.. image:: ../img/modin_in_the_cloud_architecture.png
```

Comment:
The rectangles on the diagram cross each other somewhat strange - head node rectangle should be contained within "remote environment" rectangle while it now crosses the borders which might confuse the reader. Also I suggest adding a wide rectangle depicting SSH tunnel in which put the "RPyC call" arrow.